cmd/ocm/cluster/list/list.go

```diff
@@ -112,24 +117,25 @@ func run(cmd *cobra.Command, argv []string) error {
 	// Get the client for the resource that manages the collection of clusters:
 	collection := connection.ClustersMgmt().V1().Clusters()
 
-	if cmd.Flags().Changed("managed") {
-		managed = args.managed
-	} else {
-		managed = false
-	}
-
 	// If there is a parameter specified, assume its a filter:
 	var argFilter string
 	if len(argv) == 1 && argv[0] != "" {
 		argFilter = fmt.Sprintf("(name like '%%%s%%' or id like '%%%s%%')", argv[0], argv[0])
 	}
 
 	// Update our column name and padding variables:
+	paddingByColumn := []int{34, 40, 60, 20}
+
+	// Print console url if wide parameter specified
+	if args.wide {
```

Comment:
This arg should probably conflict with manually specifying columns?

Reply:
This is a good point. Alternatively we might want to allow adding columns to the wide view? The only difference between `wide=true` and `wide=false` is the console URL.

What's the use case for this change? It might help to narrow down some of the changes. You can always do `ocm cluster list --columns=id,name,console.url`, for example

Reply:
yes, the purpose for the change is to show the console url to the output.

Currently, with the google auth enabled for the clusters, we cannot use  `ocm cluster login <cluster_id>` directly, so print the console url should be a little convenient to login to the clusters.

With the `ocm cluster list --columns=id,name,console.url` the console url in the output will be truncated in most of the cases, so we still need to add the arg `--padding 90`. This will make the whole command quite hard to input. And with the `--padding 90` option it will also expand the width for the columns other than console.url which are not needed for such wider.